src/styles/new-note.scss

```diff
@@ -0,0 +1,11 @@
+.new-note__create {
+  background-color: #0288d1;
```

Comment:
Where does this constant come from? Is the "New note" button supposed to be the same color as the button for posting annotations, which can be controlled by the publisher via the "ctaBackgroundColor" setting. CC @dawariley

Reply:
If the answer to the above question is _yes_, then in the CSS we should have the default colors for when the publisher hasn't set a call-to-action color. In the template you can use the `h-brand` attribute directive to apply the colors to the button.

Reply:
Correction, this should be `h-branding` rather than `h-brand`.  Search for uses of `h-branding=` in `src/sidebar/templates` for examples.